containers/concent-api/install-concent-api.sh.j2

```diff
@@ -8,3 +8,7 @@ source "$virtualenv_dir/bin/activate"
 python -m pip install --upgrade pip
 python -m pip install --requirement "/srv/http/concent_api/requirements.lock"
 python -m pip install gunicorn=="{{ gunicorn_version }}"
+(
+    cd "/srv/http/middleman_protocol/"
+    python setup.py develop
+)
```

Comment:
Offtopic: we should be clearing pip cache after installation in this script. You could do it in a separate pull request.

Reply:
Ok. I will make it in separate pull request.